proposals/phone-ar-depth.html

```diff
@@ -47,10 +47,10 @@
     <script id="vertexShader" type="x-shader/x-vertex">
       precision mediump float;
 
-      attribute vec2 aVertexPosition;
-      attribute float aDepthDistance;
+      in vec2 aVertexPosition;
```

Comment:
While you're making changes to this file, can you add the `depthSensing` dict to the options on line 104?

Reply:
done